cashu/mint/migrations.py

```diff
@@ -156,3 +156,23 @@ async def m007_proofs_and_promises_store_id(db: Database):
     await db.execute(
         f"ALTER TABLE {table_with_schema(db, 'promises')} ADD COLUMN id TEXT"
     )
+
+
+async def m008_paid_invoices_and_proofs(db: Database):
+    """
+    Table to store invoices paid by the mint and corresponding proofs.
+    """
+    await db.execute(f"""
+                CREATE TABLE IF NOT EXISTS {table_with_schema(db, 'payments')} (
+                    invoice_amount {db.big_int} NOT NULL,
```

Comment:
Storing an amount as a big_int, but it is converted from `msats` to `sats` obove with `/ 1000`, so any sub-sat amount is going to be truncated off... Probably OK, but a potential source of problems?